server/monitor.go

```diff
@@ -2709,6 +2710,13 @@ func (s *Server) accountDetail(jsa *jsAccount, optStreams, optConsumers, optCfg
 	}
 	jsa.mu.RUnlock()
 
+	if limits != nil {
```

Comment:
But why doing all that outside of the lock, that was my point? Sure you capture `limits` under the lock but then the lookup is done outside of the lock which can still produce races. I would do all that under the jsa.usageMu's lock at line 2703.

Reply:
yeah that makes more sense. thanks, just fixed in the next commit.